mowgli_etl/pipeline/wdc/parsimonious_parser/offers_corpus_english_v2_1000_parsed.jsonl

```diff
@@ -2,50 +2,136 @@
     "source": "offers_corpus_english_v2_1000.jsonl",
     "dimensions": [
         {
-            "depth": {
-                "value": null,
-                "unit": null
+            "weight": {
+                "value": "4",
+                "unit": "oz"
+            },
+            "line": 16,
+            "field": "description",
+            "raw_text": "westland exact fit boat covers sunbrella each westland cover is patterened to account for every curve to create an exact fit 10 year warranty 100 solution dyed acrylic 9 1 4 oz premium long lasting fabric in fade proof colors breathable water repellent mildew resistant fabric"
+        },
+        {
+            "weight": {
+                "value": "50",
+                "unit": "g"
+            },
+            "line": 24,
+            "field": "spec_table_content",
+            "raw_text": "handling references images application details target details back to top handling show hide format liquid concentration 1 g l buffer aqueous buffered solution containing 1 bsa 50 glycerol and 0 09 sodium azide preservative sodium azide precaution of use this product contains sodium azide a poisonous and hazardous substance which should be handled by trained staff only storage 20 c storage comment store at 20 c expiry date 12 months"
+        },
+        {
+            "weight": {
```

Comment:
This is "dl380 g6". You're going to need to look for whitespace before and after.

Reply:
If you output these to a spreadsheet (weight_value, weight_unit, depth_value, ..., raw_text) this might be easier to eyeball than JSON.